test/cypress/stubs/gateway-account-stubs.js

```diff
@@ -196,35 +198,41 @@ function patchAccountEmailCollectionModeSuccess (opts) {
 function patchAccountEmailCollectionModeSuccessByServiceIdAndAccountType (serviceId, accountType, emailCollectionMode = 'MANDATORY') {
   const path = `/v1/api/service/${serviceId}/account/${accountType}`
   return stubBuilder('PATCH', path, 200,
-    { op: 'replace', path: 'email_collection_mode', value: emailCollectionMode })
+    { request: { op: 'replace', path: 'email_collection_mode', value: emailCollectionMode } })
 }
 
 function setRefundEmailEnabledByServiceIdAndAccountType (serviceId, accountType, enabled) {
   const path = `/v1/api/service/${serviceId}/account/${accountType}/email-notification`
   const payload = {
-    op: 'replace',
-    path: '/refund_issued/enabled',
-    value: enabled
+    request: {
+      op: 'replace',
+      path: '/refund_issued/enabled',
+      value: enabled
+    }
```

Comment:
Interesting how this works. According to the spec the body is
```
{
  "op": "replace",
  "path": "/payment_confirmed/enabled",
  "value": false
}
```
rather than 
```
request: 
{
  "op": "replace",
  "path": "/payment_confirmed/enabled",
  "value": false
}
```

Reply:
This is just how you interact with the stub builder, this is passed in on the `additionalParams` object; where `request` is set it matches the request body against the provided object